cvmfs/receiver/receiver.cc

```diff
@@ -7,16 +7,25 @@
 #include "cvmfs_config.h"
 
 #include "../logging.h"
+#include "../monitor.h"
 #include "../swissknife.h"
+#include "../util/posix.h"
+#include "../util/string.h"
 
 #include "reactor.h"
 
+static const char *kDefaultReceiverLogDir = "var/log/cvmfs_receiver/";
```

Comment:
Shouldn't this be `/var/log/cvmfs_receiver`?

Reply:
Of course! Thanks!